fixtures/http-bis.csv

```diff
@@ -0,0 +1,4572 @@
+"binary","filename","configuration","rate","time"
```

Comment:
This CSV has not been added to `node-benchmark-compare.test.js` because it is statistically invalid, as it is missing binaries (old/new) and runs.